src/components/ly-course-list.vue

```diff
@@ -89,12 +89,29 @@ Dependencies:
         </div>
 
         <div class="col-3">
-          <a
+          <!--          <a
             class="btn indicated-btn"
             @click="decideButtonAction(course)"
           >
             {{ y18n('courseList.start') }}
             <i class="fas fa-arrow-right"></i>
+          </a>-->
```

Comment:
commenting this method out renders the content warning useless (since `decideButtonAction` is never called.)

Suggestion: Keep it in and render text and icon depending on enrollment status